src/database.rs

```diff
@@ -218,7 +218,7 @@ impl Database {
                     .unwrap()
         });
 
-        return Ok(entries);
```

Comment:
the last expression is the one returned, so this explicit return can be simpler.